class/student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom)
+    super
+    @classroom = classroom
+  end
```

Comment:
It is a requirement that the constructor extends the parent constructor (person) by adding a classroom and a parameter for it, I can see you have implemented the classroom parameter but let's make some adjustments to properly extend the parent constructor as required😉.

```suggestion
  def initialize(age, classroom, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @classroom = classroom
  end
```

You can take a look at the suggestion above and make the changes where necessary👍🏼.